contrib/node/src/python/pants/contrib/node/subsystems/node_distribution.py

```diff
@@ -233,3 +274,36 @@ def yarnpkg_command(self, args, node_paths=None):
     """
     return self._command_gen(
       [self.install_node, self.install_yarnpkg], 'yarnpkg', args=args, node_paths=node_paths)
+
+  def _configure_eslinter(self, bootstrapped_support_path):
+    logger.debug('Copying {setupdir} to bootstrapped dir: {support_path}'
+                           .format(setupdir=self._eslint_setupdir, support_path=bootstrapped_support_path))
+    safe_rmtree(bootstrapped_support_path)
+    shutil.copytree(self._eslint_setupdir, bootstrapped_support_path)
+    return True
+
+  def fetch_eslint_supportdir(self):
+    """ Returns the path where the ESLint is bootstrapped.
+
+    :returns: The path where ESLint is bootstrapped and whether or not it already installed
+    :rtype: (string, bool)
+    """
+    bootstrap_dir = os.path.realpath(os.path.expanduser(self._pants_bootstrapdir))
+    # TODO(nsaechao): ./pants clean-all should handle removing this directory.
```

Comment:
I don't think that the supportdir should be exposed outside of `.pants.d`... probably you should have the eslint subsystem hardcode this directory as `os.path.join(self.get_options().pants_workdir, 'eslint', 'support'` or something (ie, under `.pants.d`), and remove the `eslint_supportdir` option.

That would solve the clean-all problem, and would prevent people from depending on the location where eslint is installed.

Reply:
IIRC, anything in the pants_workdir gets wiped out after every pants command? Having to re-install eslint each time would be costly if this is true...

Reply:
No, only after `./pants clean-all`.